### ISSUE TITLE
Minutes is truncated

### ISSUE THREAD

**Issue by RandomSort** [eficode/pipeline-the-game#258]
Nicolaj noticed that the place holder in the card truncates minutes in a not so pretty way. See screenshot.

Perhaps it would be better as `15m` or `15 mins`.
![image](https://user-images.githubusercontent.com/582571/111757522-b309bb80-889b-11eb-99f5-95d572d747ac.png)

**Comment by sofusalbertsen** [comment]
Yes, during development, when asked about what would be plausible to type there people wrote all kinds of things like "4 minutes more or less" Or "it depends", so we can't rely that people only type metrics that are computer parseable in.
Do you have a suggestion for how to handle that?

**Comment by rams23** [comment]
@sofusalbertsen in other projects we used a text parser from Microsoft ([here](https://github.com/microsoft/Recognizers-Text)) to get parsed time units from a text. 
Either you can enforce the input with a number and a time unit, or if you want to let it free, we can use a library like that one to parse a time interval into a standard representation.
